stdlib/_json.pyi

```diff
@@ -3,14 +3,22 @@ from typing_extensions import final
 
 @final
 class make_encoder:
-    sort_keys: Any
-    skipkeys: Any
-    key_separator: Any
-    indent: Any
-    markers: Any
-    default: Any
-    encoder: Any
-    item_separator: Any
+    @property
+    def sort_keys(self) -> Any: ...
+    @property
+    def skipkeys(self) -> Any: ...
+    @property
+    def key_separator(self) -> Any: ...
+    @property
+    def indent(self) -> Any: ...
+    @property
+    def markers(self) -> Any: ...
+    @property
+    def default(self) -> Any: ...
+    @property
+    def encoder(self) -> Any: ...
+    @property
+    def item_separator(self) -> Any: ...
```

Comment:
```suggestion
    @property
    def sort_keys(self): ...
    @property
    def skipkeys(self): ...
    @property
    def key_separator(self): ...
    @property
    def indent(self): ...
    @property
    def markers(self): ...
    @property
    def default(self): ...
    @property
    def encoder(self): ...
    @property
    def item_separator(self): ...
```

Reply:
Actually, why don't I just add the annotations as part of this PR, I believe they're just the same as in the `__init__` function.